### DISCUSSION TITLE
I built an open-source CLI tool to ingest/copy data between any databases

### DISCUSSION THREAD

**Discussion by denizhdz** [community/community#110315] [Programming Help]
### Select Topic Area

Show & Tell

### Feature Area

Projects

### Body

Hi all, ingestr is an open-source command-line application that allows ingesting & copying data between two databases without any code: https://github.com/bruin-data/ingestr

It does a few things that make it the easiest alternative out there:

- ✨ copy data from your Postgres / MySQL / SQL Server or any other source into any destination, such as BigQuery or Snowflake, just using URIs
- ➕ incremental loading: `create+replace`, `delete+insert`, `append`
- 🐍 single-command installation: `pip install ingestr`

We built ingestr because we believe for 80% of the cases out there people shouldn’t be writing code or hosting tools like Airbyte just to copy a table to their DWH on a regular basis. ingestr is built as a tiny CLI, which means you can easily drop it into a cronjob, GitHub Actions, Airflow or any other scheduler and get the built-in ingestion capabilities right away.

Some common use-cases ingestr solve are:

- Migrating data from legacy systems to modern databases for better analysis
- Syncing data between your application's database and your analytics platform in batches or incrementally
- Backing up your databases to ensure data safety
- Accelerating the process of setting up new environment for testing or development by easily cloning your existing databases
- Facilitating real-time data transfer for applications that require immediate updates

We’d love to hear your feedback, and make sure to give us a star on GitHub if you like it! 🚀 https://github.com/bruin-data/ingestr

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi @denizhdz , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion, asking users to star their repository, or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
